README.md

```diff
@@ -1,22 +1,24 @@
-# fun
+# Fun
 
 (have)Fun with Serverless
 
 ![logo.jpg](./logo.jpg)
 
-[fun](https://github.com/aliyun/fun) 是一个用于支持 Serverless 应用部署的工具，能帮助您便捷地管理函数计算、API 网关、日志服务等资源。它通过一个资源配置文件（template.yml），协助您进行开发、构建、部署操作。
+[中文文档](README-zh.md)
 
-如果您想要使用旧版语法，请[参考](./README.old.md)。
+[Fun](https://github.com/aliyun/fun) is a development tool for serverless applications. It could help you efficiently arrange cloud resources such as Function Compute, API Gateway, Log Service and so on. You can use it to develop，build and deploy FC by describing relative resources in a `template.yml` file.
 
-## 安装
+If you want to use the old syntax, please refer to [README.old.md](./README.old.md).
 
-fun 是一个 Node.js 编写的命令行工具，但它也能支持 Python，Java 等环境的部署操作。安装它的方式是通过 npm：
+## Installation
+
+Fun is a command-line tool written in Node.js, but it can also support the deployment of Python, Java and other environments. The easiest way to install `Fun` is to use [npm](https://www.npmjs.com/):
 
 ```
 $ npm install @alicloud/fun -g
 ```
 
-安装完成之后，会有一个 fun 命令提供使用。输入 fun 命令可以查看帮助信息：
+After the installation is complete, a `fun` command is provided for use. Enter `fun` command to see help information:
```

Comment:
A `fun` command is available after installed completely. Type `fun` in the console will print usage: